RELEASE_NOTES.md

```diff
@@ -2,38 +2,83 @@
-  - `ReceiveElectricalComponentDataStream` RPC method has been renamed to `ReceiveElectricalComponentTelemetryStream` to match the naming conventions used in `frequenz-api-common`.
-  - `ReceiveElectricalComponentDataStreamRequest` RPC method has been renamed to `ReceiveElectricalComponentTelemetryStreamRequest` to match the naming conventions used in `frequenz-api-common`.
-  - `ReceiveElectricalComponentDataStreamResponse` RPC method has been renamed to `ReceiveElectricalComponentTelemetryStreamResponse` to match the naming conventions used in `frequenz-api-common`.
-  - `ReceiveSensorDataStream` RPC method has been renamed to `ReceiveSensorTelemetryStream` to match the naming conventions used in `frequenz-api-common`.
-  - `ReceiveSensorDataStreamRequest` RPC method has been renamed to `ReceiveSensorTelemetryStreamRequest` to match the naming conventions used in `frequenz-api-common`.
-  - `ReceiveSensorDataStreamResponse` RPC method has been renamed to `ReceiveSensorTelemetryStreamResponse` to match the naming conventions used in `frequenz-api-common`.
-  - `ListComponentConnectionRequests.starts` has been renamed to `ListElectricalComponentConnectionRequests.source_electrical_component_ids` to match the naming conventions used in `frequenz-api-common`.
-  - `ListComponentConnectionRequests.ends` has been renamed to `ListElectricalComponentConnectionRequests.destination_electrical_component_ids` to match the naming conventions used in `frequenz-api-common`.
-- The RPC `GetMicrogridMetadata` has been renamed to `GetMicrogrid` to better align with the naming convention in `frequenz-api-common`.
-- The `GetMicrogridMetadataResponse` message has been renamed to `GetMicrogridResponse` to better align with the naming convention in `frequenz-api-common`.
-- The response messages for the `SetElectricalComponentPowerActive` and
-`SetElectricalComponentPowerReactive` RPCs have been changed to return a stream of responses instead of a single response. This allows the server to provide ongoing status updates for the request, which is useful when the electrical component cannot immediately set its output power to the requested value.
-- The RPC `AddComponentBounds` has been renamed to `AugmentElectricalComponentBounds`.
-- `SetElectricalComponentPowerActive` and `SetElectricalComponentPowerReactive` RPCs ave been replaced by a new RPC `SetElectricalComponentPower`. The new RPC allows setting either the active or reactive power of an electrical component, depending on the value of the `power_type` field in the request message.
-
-## New Features
-
-<!-- Here goes the main new features and examples or instructions on how to use them -->
-
-## Bug Fixes
-
-<!-- Here goes notable bug fixes that are worth a special mention or explanation -->
+### 1. Dependency Updates
+
+Your project's dependencies may need to be updated.
+
+  * The minimum supported version of `protobuf` is now **`5.28.1`**.
+  * The minimum supported version of `grpcio` is now **`1.66.1`**.
```

Comment:
Not for this PR, but there are newer versions for these, maybe if we are going for a new "major" release here we should bump these to the latest available?

Reply:
https://github.com/frequenz-floss/frequenz-api-microgrid/issues/343 (I have assigned it to you for now)

Reply:
Done.